### PR TITLE
Command to pack audio files and prepare release on Zenodo #42

### PR DESCRIPTION
**Pull request by Bibiko**
+ made contributors/creators unique in metadata.get_creators_and_contributors

### CODE REVIEW

File: src/cldfbench/commands/media.py [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+
+import collections
+import html
+import mimetypes
+import os
+import pathlib
+import subprocess
+import threading
+import time
+import tqdm
```
Comment (xrotwang) [inline src/cldfbench/commands/media.py:38]: `tqdm` should be down in the block of third-party packages, like our `zenodoclient`.

File: src/cldfbench/commands/media.py [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+from clldutils.clilib import PathType
+from clldutils.misc import format_size, nfilter
+from clldutils.path import md5, git_describe
+from csvw.dsv import UnicodeWriter
+from zenodoclient.api import Zenodo, API_URL, API_URL_SANDBOX, ACCESS_TOKEN
+from zenodoclient.models import PUBLISHED
+
+
+MEDIA = 'media'
+ZENODO_FILE_NAME = 'zenodo.json'
```
Comment (xrotwang) [inline src/cldfbench/commands/media.py:57]: Why not `.zenodo.json`?

File: src/cldfbench/commands/media.py [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+COMMUNITIES = ['lexibank']
+LISENCE = 'This dataset is licensed under {0}.'
```
Comment (xrotwang) [inline src/cldfbench/commands/media.py:59]: LI**C**ENCE

File: src/cldfbench/commands/media.py [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+
+    while threading.active_count() > 7:
+        time.sleep(0.1)
+
+    download_thread = threading.Thread(target=_download, args=(url, target))
+    download_thread.start()
+    download_threads.append(download_thread)
+
+
+def _git_repo_url(dir_, git_command='git'):
```
Comment (xrotwang) [inline src/cldfbench/commands/media.py:147]: Can be replaced with
```python
[rem for rem in ds.repo.repo.remotes if rem.name == 'origin'][0].url.replace('.git', '')
```

File: src/cldfbench/commands/media.py [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+    return res.replace('.git', '')
+
+
+def run(args):
+
+    ds = get_dataset(args)
+    ds_cldf = ds.cldf_reader()
+    release_dir = args.out / '{0}_{1}'.format(ds.id, MEDIA)
+
+    if ds_cldf.get('media.csv', None) is None:
```
Comment (xrotwang) [inline src/cldfbench/commands/media.py:173]: I think for all these preliminary input checks, I'd raise [`ParserError`](https://github.com/clld/clldutils/blob/67a8c493c2bbf37c107ebae72bbe80e8d6a2a1a3/src/clldutils/clilib.py#L21).

**Review by xrotwang — CHANGES_REQUESTED** [review]

File: src/cldfbench/commands/media.py (hunk 2 above)
Comment (Bibiko) [reply src/cldfbench/commands/media.py:57]: per default it would be an hidden file and acc. to workflow one should check it before usage, thus it's easier

Comment (xrotwang) [reply src/cldfbench/commands/media.py:57]: Ok

**Review by xrotwang — APPROVED** [review]

**Review by xrotwang — APPROVED** [review]

**Pull request merged**